docker-compose.hobbies.yml

```diff
@@ -54,6 +54,22 @@ services:
         - APP_PORT=3000
         - PROJECT=hobbies-helsinki
 
+  # Convenience production "cache" stage service for debugging
```

Comment:
What about the other docker-compose files?

Reply:
Cache image support added  to sports and events docker-compose too